pointillism-port.patch

```diff
@@ -0,0 +1,129 @@
+diff --git a/activity.py b/activity.py
+index d91ea44..c4edf8b 100755
+--- a/activity.py
++++ b/activity.py
+@@ -1,4 +1,4 @@
+-#! /usr/bin/env python
++#! /usr/bin/env python3
```

Comment:
Can you move this to a separate patch ? so we can send it upstream later? e.g. pointillism-fix-shebang.patch

Reply:
Shall I just remove the shebang and follow only python for now? Basically we don't execute any files without adding the python or python3 before the file uri